motoman/0829607/0829607.bug

```diff
@@ -28,11 +28,12 @@ bug:
   reproducibility: always
   trace: null
 fix:
-  repo: https://github.com/gavanderhoorn/motoman
-  hash: 57b7a995dfdcf025d6394123eb87896853a6aadf
   pull-request: null
   license: null
   fix-in: ['motoman_sda10f_support/meshes/sda10f/visual/motoman_axis_b1.stl', 'motoman_sda10f_support/meshes/sda10f/visual/motoman_base.stl']
   languages:
     - STL
   time: 2016-08-29 (13:15)
+  commits:
+    - repo: https://github.com/gavanderhoorn/motoman
```

Comment:
Just noticed this: why are we linking to a fork of `motoman` for the fix?

It would make more sense to me to only see `ros-industrial/motoman` here, as that should contain the fix commits as well.

We don't have any forks of these forks in `robust-rosin`, which I would expect we'd need to do if we start linking to forks of the repositories we're including in ROBUST.